shed/event_streams.py

```diff
@@ -929,6 +929,8 @@ def __init__(self, func, child, state_key=None,
                              output_info=output_info)
         if len(input_info) != 1:
             raise ValueError("Error : only one key allowed for accumulate")
+        # only one input expected
+        self.input_key = list(self.input_info)[0]
```

Comment:
Can we have the option for functions which have no second argument? (eg a counter)

Reply:
sounds good, but we'll have to enforce that a start state is defined and not just `no_default` (since there is no more input to pull from).
Also, I personally like an option allowing the function to also return its state as [here](https://github.com/mrocklin/streamz/blob/master/streamz/core.py#L483). Do you agree with this? I can do this later in the afternoon. (Will need to add extra tests)

Reply:
I am good with having the function return the initial state as well, although we should write a helper decorators for that since many functions don't do that by themselves.